pkg/katautils/config.go

```diff
@@ -700,6 +700,96 @@ func newAcrnHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 	}, nil
 }
 
+func newIchHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
+	hypervisor, err := h.path()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
+	}
+
+	kernel, err := h.kernel()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
+	}
+
+	initrd, image, err := h.getInitrdAndImage()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
+	}
+
+	if image != "" && initrd != "" {
+		return vc.HypervisorConfig{},
+			errors.New("having both an image and an initrd defined in the configuration file is not supported")
+	}
+
+	if image == "" && initrd == "" {
```

Comment:
I am not sure but I think CH does not support initrd today, just via https://github.com/intel/rust-hypervisor-firmware I am not sure will need in the future  initrd support  or just use initrd, but I'd say we  just limit image base boot for now @egernst any comments?

Reply:
changed initrd and image check accordingly.